Microsoft.Toolkit.Uwp.UI.Controls.Core/InAppNotification/Styles/MSEdgeNotificationStyle.xaml

```diff
@@ -150,9 +151,10 @@
                         FontFamily="Segoe MDL2 Assets"
                         FontSize="12"
                         VerticalAlignment="Top"
+                        Padding="4"
                         Style="{StaticResource DismissTextBlockButtonStyle}">
                     <Button.RenderTransform>
-                        <TranslateTransform x:Name="DismissButtonTransform" X="25" Y="-5"/>
+                        <TranslateTransform x:Name="DismissButtonTransform" X="20" Y="2"/>
```

Comment:
You are using `Y="1"` in the `.bind`. I guess we should have the same value in the two files ?

Reply:
@XAML-Knight any reason for this?

Reply:
I went with what looked good, visually.  Would be nice to have a consistent layout, even though the difference is only 1 pixel between the two button transforms.